packages/icons/src/HarnessIcons.ts

```diff
@@ -3047,6 +3049,7 @@ const HarnessIcons: KVO<ElementType> = {
   dashboard: Dashboard,
   'dashboards-solid-border': DashboardsSolidBorder,
   'data-fetch-error': DataFetchError,
+  'db-devops': DbDevops,
```

Comment:
This logo is yet not final. 
For now, we can merge the PR with this one. I will change the svg in case if the logo changes